crates/ev-cli/src/commands/relay/create.rs

```diff
@@ -0,0 +1,124 @@
+use crate::CmdOutput;
+use clap::Parser;
+use common::{
+    api::{
+        client::ApiError,
+        papi::{self, EvApi},
+        BasicAuth,
+    },
+    relay::Relay,
+};
+use std::path::PathBuf;
+use thiserror::Error;
+
+use crate::commands::interact::{self, validated_input};
+/// Creates an Evervault Relay and generates its configuration file
+#[derive(Parser, Debug)]
+#[command(name = "create")]
+pub struct CreateArgs {
+    /// Path to write relay.json to. Defaults to relay.json
+    #[arg(short = 'o', long = "out", default_value = "relay.json")]
+    pub out: String,
+    #[arg(short = 'f', long = "force", default_value = "false")]
+    pub force: bool,
+}
+
+#[derive(Error, Debug)]
+pub enum CreateError {
+    #[error(
+        "A Relay configuration file already exists at the path: {0}, use the --force parameter to overwrite the existing file"
+    )]
+    FileAlreadyExists(String),
+    #[error("An error occured while writing the Relay configuration file:\n{0}]")]
```

Comment:
We're not using the line break after our message in `RelayConfigError`, should be consistent (don't think the line break is necessary)